apps/user-office-frontend/src/components/call/CallGeneralInfo.tsx

```diff
@@ -321,26 +336,54 @@ const CallGeneralInfo: React.FC<{
           data-cy="reference-number-format"
         />
       </LocalizationProvider>
-
-      <FormikUIAutocomplete
-        name="templateId"
-        label="Call template"
-        loading={loadingTemplates}
-        noOptionsText="No templates"
-        items={templateOptions}
-        InputProps={{ 'data-cy': 'call-template' }}
-        required
-      />
-      {featuresMap.get(FeatureId.RISK_ASSESSMENT)?.isEnabled && (
+      <FormControl fullWidth>
         <FormikUIAutocomplete
-          name="esiTemplateId"
-          label="ESI template"
+          name="templateId"
+          label="Call template"
           loading={loadingTemplates}
           noOptionsText="No templates"
-          items={esiTemplateOptions}
-          InputProps={{ 'data-cy': 'call-esi-template' }}
+          items={templateOptions}
+          InputProps={{ 'data-cy': 'call-template' }}
           required
         />
+        <Link
+          href={
+            templateId ? `QuestionaryEditor/${templateId}` : 'ProposalTemplates'
+          }
+          target="_blank"
+          className={classes.textRightAlign}
+        >
+          View/Edit Call templates
```

Comment:
Should the title also change based on if the template is selected or not?

Reply:
Good point, I'll try it out tomorrow